src/coreclr/src/debug/daccess/nidump.cpp

```diff
@@ -6766,11 +6766,11 @@ NativeImageDumper::DumpMethodTable( PTR_MethodTable mt, const char * name,
         MethodTableToString( mt, buf );
         m_display->ErrorPrintF( "WARNING! MethodTable %S is generic but is not hard bound to its EEClass.  Cannot compute generic dictionary sizes.\n", (const WCHAR *)buf );
     }
-    else if( !m_isMscorlibHardBound )
+    else if( !m_isCoreLibHardBound )
     {
         /* REVISIT_TODO Mon 8/20/2007
-         * If we're not hard bound to mscorlib, most things don't work.  They depend on knowing what
-         * g_pObjectClass is.  Without the hard binding to mscorlib, I can't figure that out.
+         * If we're not hard bound to CoreLib, most things don't work.  They depend on knowing what
```

Comment:
"* REVISIT_TODO Mon 8/20/2007" 😊

Reply:
This file has a lot of these very old TODO comments...

Reply:
Examples like this is why I prefer we don't store TODO's in product code, but I know there's no consensus about it.